floodviz/map_utils.py

```diff
@@ -20,8 +38,19 @@ def site_dict(site_list, url_prefix):
     # get data from url
     req = requests.get(url)
 
+    if not req.text.startswith('#'):
+        print("Error: request returns invalid data")
+        return
+
     # put data into list
-    data = req.text.splitlines()[29:]
+    data = []
+    for line in req.text.splitlines():
+        if not line.startswith('#'):
+            data.append(line)
+
+    if not data:
```

Comment:
It's possible that your call won't return any data. It's not an error so this should not print out an error and instead return an empty array.